database/migration/migrateData.ts

```diff
@@ -854,6 +856,83 @@ const migrateData = async () => {
     }
   }
 
+  // v0.8.0
+  if (databaseVersionLessThan('0.8.0')) {
+    console.log('Migrating to v0.8.0...')
+
+    console.log(' - Creating permission_flattened view')
+    await DB.changeSchema(`
+        CREATE INDEX IF NOT EXISTS "i_review_status_history_status" ON review_status_history (status);
```

Comment:
I also tried this with a table, where user_id, organisation_id and policy_id are indexed (individually and as one index), and it was around 1.2x, which is small improvement but requires a bit of maintenance of the table (I had it refreshing for a user when they log in or change or, i was worried that indexes might get slower the more we drop and re-insert rows, and it's more moving parts, so just went with a view)

Reply:
Can you put all views etc in `43_views_functions_triggers.sql` -- we drop all of them and re-create them every time, which prevents a lot of dependency problems, and is super fast. Save the migration script for actual schema changes or data migrations.